lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.h

```diff
@@ -282,6 +284,13 @@ class SymbolFileNativePDB : public SymbolFileCommon {
       m_parent_types;
 
   lldb_private::UniqueCStringMap<uint32_t> m_type_base_names;
+
+  /// Global ID -> mangled name/full function name
```

Comment:
Did you mean for these mapping comments to be reversed. E.g.,:
```
/// mangled name/full function name -> Global ID
```
?

Reply:
Right, sorry for the confusion.